File: Recent-Tweet-Counts/recent_tweet_counts.go
package Recent_Tweet_Counts

// Search for Tweets within the past
// https://developer.twitter.com/en/docs/twitter-api/tweets/search/quick-start/recent-search

import (
	"encoding/json"
	"fmt"
	"log"
	"net/http"
	"os"
	"time"
)

type RecentTweets struct {
	Data []Data `json:"data"`
	Meta Meta   `json:"meta"`
}
type Data struct {
	End        time.Time `json:"end"`
	Start      time.Time `json:"start"`
	TweetCount int       `json:"tweet_count"`
}
type Meta struct {
	TotalTweetCount int `json:"total_tweet_count"`
}

func RecentTweetCount() {
	if os.Getenv("BEARER_TOKEN") == "" {
		log.Fatalln("set a bearer token")
	}

	const url = "https://api.twitter.com/2/tweets/counts/recent"

	// Sets the bearer token from your environment variables
	// To set environment variables on macOS or Linux, run the export command below from the terminal:
	// export BEARER_TOKEN='YOUR-TOKEN'
	bearerToken := os.Getenv("BEARER_TOKEN")
	client := http.Client{}

	request, err := http.NewRequest("GET", url, nil)
	if err != nil {
		log.Println(err)
	}
	// Add Bearer Token to HTTP Header
	request.Header.Set("Authorization", "Bearer "+bearerToken)

	// Add Paramater
	q := request.URL.Query()
	q.Add("query", "from:TwitterDev")
	q.Add("granularity", "day")
	request.URL.RawQuery = q.Encode()

	// Send Request
	response, err := client.Do(request)
	if err != nil {
		log.Println(err)
	}

	defer response.Body.Close()

	var recentTweet RecentTweets
	err = json.NewDecoder(response.Body).Decode(&recentTweet)
	if err != nil {
		log.Println(err)
	}

	fmt.Println(recentTweet)
}

File: Recent-Tweet-Counts/recent_tweet_counts.php
<?php

/**
 * To use these samples you will need to set your bearer token as an environment variable by running the following in your terminal:
 * export BEARER_TOKEN='<your_bearer_token>'
 */

const RECENT_TWEET_COUNTS_API_URL = 'https://api.twitter.com/2/tweets/counts/recent';

/**
 * @throws Exception
 */
function get_recent_tweet_counts(array $headers, array $params = []): array
{
    $url = RECENT_TWEET_COUNTS_API_URL.'?'.http_build_query($params);

    $curl = curl_init();
    curl_setopt($curl, CURLOPT_URL, $url);
    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
    curl_setopt($curl, CURLOPT_HTTPHEADER, $headers);
    $response = curl_exec($curl);
    $status_code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
    curl_close($curl);

    $decoded_response = json_decode($response, true);

    if ($status_code != 200) {
        $errors = print_r($decoded_response['errors'] ?? $decoded_response, true);

        throw new Exception("Unable to get tweet counts: ".$errors, $status_code);
    }

    return $decoded_response['data'] ?? [];
}

try {
    $headers = [
        'Authorization: Bearer '.getenv('BEARER_TOKEN')
    ];

    // update the array below with your parameters:
    // https://developer.twitter.com/en/docs/twitter-api/tweets/counts/api-reference/get-tweets-counts-recent
    $params = [
        'query' => 'from:twitterdev',
        'granularity' => 'day'
    ];

    $tweet_counts = get_recent_tweet_counts($headers, $params);
} catch (Exception $e) {
    print $e->getMessage();
}
